Snippets/ASBFunctions/ASBFunctions_1.3/FunctionsHostBuilderUsage.cs

```diff
@@ -0,0 +1,141 @@
+using NServiceBus;
+
+// needs to be top level first defined for C# to compile
+#region endpoint-trigger-function-wireup
+
+[assembly: NServiceBusTriggerFunction("MyFunctionsEndpoint",
+    TriggerFunctionName = "NServiceBusFunctionEndpointTrigger-ASBTriggerQueue")]
+
+#endregion
+
+namespace ASBFunctions_1_3_Autogenerated
+{
+    #region asb-function-hostbuilder-trigger
+    // <autogenerated/>
+    using System.Threading.Tasks;
+    using Microsoft.Azure.ServiceBus;
+    using Microsoft.Azure.ServiceBus.Core;
+    using Microsoft.Azure.WebJobs;
+    using Microsoft.Extensions.Logging;
+    using NServiceBus;
+
+    class FunctionEndpointTrigger
+    {
+        readonly IFunctionEndpoint endpoint;
+
+        public FunctionEndpointTrigger(IFunctionEndpoint endpoint)
```

Comment:
that's not what the new generator generates, as it uses `FunctionEndpoint` directly. Not sure that's something we should show as we want users to use the `IFunctionEndpoint` for manual usage. Do we even need to show the generated code at all?

Reply:
removed the auto-genned section from the snippet.